test/quantization/test_quantized_op.py

```diff
@@ -2703,23 +2703,77 @@ def test_qlinear_unpack(self, W, use_channelwise):
                 W_q.q_zero_point(), W_q_origin.q_zero_point())
 
 
-# This testcase tests the underlying operators that are implemented in C++.
 @unittest.skipIf(sys.platform == "darwin", "Known test failure on Mac.")
 class TestQuantizedEmbeddingBag(TestCase):
+    def _test_embedding_bag_unpack_fn(self, pack_fn, unpack_fn, num_embeddings, embedding_dim, bit_rate):
+        weights = torch.from_numpy((np.random.random_sample((
+            num_embeddings, embedding_dim)) + 1).astype(np.float32))
+        w_packed = pack_fn(weights)
+        w_unpacked = unpack_fn(w_packed)
+
+        # compare against C2 to ensure numerical equivalency.
+        from caffe2.python import core, workspace
+        conversion_op = "FloatToFused8BitRowwiseQuantized"
+        if bit_rate == 4:
+            conversion_op = "FloatToFused4BitRowwiseQuantized"
+
+        def get_c2_weights(weights):
+            workspace.ResetWorkspace()
+
+            workspace.FeedBlob("weights", weights)
+            workspace.RunOperatorOnce(
```

Comment:
Using C2 operators gives confidence that PT ops work. Do you plan to keep the comparison with C2 for the longer term as well?

Reply:
No, we will add reference implementations eventually once C2 is deprecated.